### ISSUE TITLE
macOS Safari displays burger icon als question mark

### ISSUE THREAD

**Issue by reinhardmohr** [ampproject/amphtml#13670]
## What's the issue?

The safari browser on mac displays the burger icon as a question mark. This happens on my own validated AMP page s well as on ampstart.com
Chrom displays everything properly. 
I tried to substitute the burger icon ☰ by codes like &#x2630; &#9776; but it does not help.

## How do we reproduce the issue?

1. Probably: Use Safari 11.0.3 and go to the templates on ampstart.com and set them to mobile display size.
2. You can go to my website in progress www.physiomuenchen.de and do the same – but I have just begun there and it ill frequently change.

## What browsers are affected?

Happens on Safari 11.0.3 on Mac 10.13.3
Does not happen with Firefox and Chrome on this Mac.
Does not happen on my iPhone 6s with latest iOs and Safari

## Which AMP version is affected?
https://cdn.ampproject.org/v0.js

I am just starting with AMP, so it‘s new for me.

I attach three screenshots:
Rendering by Safari, rendering by Chrome, Screenshot of source code.
Thanks for helping!
![bildschirmfoto 2018-02-26 um 19 32 57](https://user-images.githubusercontent.com/13329800/36688263-2dd34ed4-1b2c-11e8-96fb-3b45746603ff.png)
![bildschirmfoto 2018-02-26 um 19 32 15](https://user-images.githubusercontent.com/13329800/36688275-35e42e04-1b2c-11e8-8976-7878441a45bd.png)
![bildschirmfoto 2018-02-26 um 19 33 18](https://user-images.githubusercontent.com/13329800/36688282-3947545e-1b2c-11e8-93f2-6a2d902df92e.png)

**Comment by aghassemi** [comment]
@reinhardmohr Please use the `svg` version of the hamburger icon same way the "Bike Shed" template in ampstart does: https://ampstart.com/templates#commerce

**Comment by aghassemi** [comment]
Closing since this is not an AMP-specific issue and a browser issue . work-around mentioned above.

**Comment by reinhardmohr** [comment]
Hi, thanks for helping. Using the svg helped – but the closing x also rendered as a question mark in Safari. And there was no svg available.
So I looked a bit deeper. The problem was: In the HTML code the opening burger icon and the closing x were inserted from a different font. And this font obviously was not available for Safari.
Solution:
I looked in the custom css where I could find "font-family". And for the body-element I removed 

`-apple-system,`

Now Safari obviously chose from the remaining four or five fonts for body and chose the one to properly render the burger icon and the closing x.
Thanks for helping
Reinhard
